noir-projects/aztec-nr/aztec/src/authwit/auth_selector.nr

```diff
@@ -4,32 +4,32 @@ use dep::protocol_types::{
 };
```

Comment:
Could we keep the file name consistent with the struct name? Then I find it easier to navigate the code.